Bugsnag/BugsnagSessionTracker.h

```diff
@@ -48,17 +48,6 @@ static NSNotificationName const BSGSessionUpdateNotification = @"BugsnagSessionC
  */
 - (void)startNewSessionIfAutoCaptureEnabled;
 
-/**
- Update the details of the current session to account for externally reported
- session information. Current session details are included in subsequent crash
- reports.
- */
-- (void)registerExistingSession:(NSString *)sessionId
```

Comment:
This is not unused https://github.com/bugsnag/bugsnag-unity/blob/bdd771d30f635f5ead5225186a5196355b888c63/src/BugsnagUnity.m#L356

Reply:
Thanks, well spotted

Reply:
Restored and documented